src/resource.rs

```diff
@@ -1,6 +1,18 @@
 use bevy::prelude::*;
+use std::fmt;
 
 #[derive(Resource)]
 pub struct Fonts {
     pub primary: Handle<Font>,
-}
\ No newline at end of file
+}
+
+#[derive(Resource)]
+pub struct Points {
+    pub value: u32,
```

Comment:
For something like this it is best to avoid the named value and instead use
`pub struct Points(pub u32)`

Reply:
I do not like that style of struct, because when it comes time to consume the contained value you end up writing
```rust
points.0 += worth.0;
```
Which looks like ass to me. I could maybe implement deref or something to auto refer to the internal value, but that's a dangerous game.